### PR TITLE
391 - clean up doc directory

### LINKED ISSUE #391: Clean up doc directory
- [ ] move code out, things like the SQL scheme definition
- [ ] compare documentation to the readme file in /doc**s** and add whatever may be missing
- [ ] remove /doc directory

### PR DESCRIPTION
**Pull request by dorians**
closes #391

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -61,6 +61,8 @@ Both `conseil` and `lorre` write to `syslog`. The logs are verbose enough to det
 
 `conseil` and `lorre` use [HOCON](https://github.com/lightbend/config/blob/master/HOCON.md) format to store configuration. Read more at [Lightbend Config](https://github.com/lightbend/config) github page.
 
+For additional extra fine-grained configuration you can refer to [this appendix](extra-custom-config.md)
```
Comment (ivanopagano) [inline docs/README.md]: I'm generally not sure why we have a duplicate readme here?

Comment (dorians) [reply docs/README.md]: I don't get it as well. It looks like one is more technical and detailed and the second one is more in "what's this project about" style.

**Comment by dorians** [comment]
> The `doc/conseil.sql` file is referenced from the `docker-compose.yml` file in the project root, please update that file too.
> 
> I think there are no more reference to that.

@ivanopagano Good catch. Thanks! fixed

**Comment by vishakh** [comment]
@dorians This needs a pull from `master` for merging.

**Pull request merged**